app.js

```diff
@@ -0,0 +1,23 @@
+const express = require("express");
+const { getTopics } = require("./Controllers/controller.js");
+
+const app = express();
+
+app.use(express.json());
+app.get("/api/topics", getTopics);
+
+// for next endpoint
+app.use((err, req, res, next) => {
+  if (err.msg && err.status) {
+    res.status(err.status).send({ msg: err.msg });
+  } else {
+    next(err);
+  }
+});
+
```

Comment:
You are right in thinking this error handler is not relevant... YET!
It can stay here for now but only needs to be introduced alongside the relevant ticket later on